lib/Cleantalk/Antispam/Integrations/LeadFormBuilder.php

```diff
@@ -37,7 +45,12 @@ public function getDataForChecking($argument)
             $_POST['fdata'] = http_build_query($query_array, '', '&', PHP_QUERY_RFC3986);
         }
 
-        return ct_gfa($data_for_test);
+        $gfa_result = ct_gfa($data_for_test);
```

Comment:
One other way to make the code simpler: pass the email and the nickname into `ct_gfa` function via parameters
```php
 $gfa_result = ct_gfa($data_for_test, $email, $nickname);
 $gfa_result['event_token'] = !empty($event_token) ? $event_token : null;
 return $gfa_result;
```

Reply:
Agreed. Done.